SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+        feedbackData = []
+        return feedbackData.count
+        
+    }
+    
+    func setUpNavigationBar() {
+        
+        navigationController?.navigationBar.barTintColor = Constants.navigationTintColor
+        
+        let attributedTitle = NSAttributedString(string: "Feedback", attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoMedium, size: 21)!, NSAttributedString.Key.foregroundColor: Constants.titleColor])
+        title = attributedTitle.string
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "pen"), style: UIBarButtonItem.Style.plain, target: self, action: #selector(handleNavigationBarRightTap))
+        
+    }
+    
+    func setUpFeedbackLabel() {
+        
+        feedbackLabel = UILabel()
+        feedbackLabel.lineBreakMode = .byWordWrapping
+        feedbackLabel.numberOfLines = 0
+
+        // format the alignment and spacing between lines
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        
+        // format the text with specific color, font, size
+        let title = "No Feedback Yet\n"
+        let subtitle = "See feedback conversations here"
+        
+        let attributedText = NSMutableAttributedString(string: title, attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoMedium, size: 17)!, .paragraphStyle: paragraphStyle, NSAttributedString.Key.foregroundColor: Constants.titleColor])
+        
+        attributedText.append(NSAttributedString(string: subtitle, attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoRegular, size: 13)!, NSAttributedString.Key.foregroundColor: Constants.subtitleColor]))
+
+        feedbackLabel.attributedText = attributedText
+        feedbackLabel.sizeToFit()
+        view.addSubview(feedbackLabel)
+        
+    }
+    
+    func setUpConversationButton() {
+        
+        newConversationButton = UIButton()
+        newConversationButton.setTitle("Start Conversation", for: .normal)
+        newConversationButton.backgroundColor = Constants.themeColor
+        newConversationButton.titleLabel?.font = UIFont(name: Constants.robotoMedium, size: 17)
+        newConversationButton.layer.cornerRadius = 22
+        view.addSubview(newConversationButton)
+        
+    }
+    
+    func setUpFeedbackTableView() {
+        
+        feedbackTableView = UITableView(frame: .zero)
+        feedbackTableView.translatesAutoresizingMaskIntoConstraints = false
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.register(FeedbackTableViewCell.self, forCellReuseIdentifier: reuseID)
+        view.addSubview(feedbackTableView)
+                    
+    }
+    
+    func setUpConstraints() {
+                
+        if isFeedback {
```

Comment:
If im understanding this correctly, `isFeedback` is true when there exists feedback already? And the else case is to handle the empty state. Two comments: 
let's change `isFeedback` to either be a function that just checks `feedbackData.isEmpty()` or just replace all calls with that same function call. 
Also for handling empty state in tableviews, there's this cool pod we tend to use across several of our apps: https://github.com/dzenbot/DZNEmptyDataSet is pretty good with this kinda case and removes a lot of the "if empty handle this else" kinda code